src/lib/timedateutils.ts

```diff
@@ -43,6 +43,16 @@ export const formatDateRangeShort = (startDate: Date, endDate: Date) => {
     : `${startDateString} - ${endDateString}`;
 };
 
+export const isPastDeadline = (
+  dateToCheck: Date,
+  deadline: Date | null
+): boolean => {
+  if (deadline) {
+    return dateToCheck.getTime() > deadline.getTime();
```

Comment:
Do we ever have a scenario where `dateToCheck` is some other than the current date? If not maybe this function could be changed to always check against the current date e.g.;
```suggestion
    return new Date().getTime() > deadline.getTime();
```

Reply:
You're right, can't think of such a scenario, so I'll simplify this according to your suggestion.